src/lib/network_pool/transaction_pool.ml

```diff
@@ -166,6 +170,9 @@ module Diff_versioned = struct
           "The diff containing this transaction was too large"
       | Fee_payer_account_not_found ->
           "Fee payer account was not found in the best tip ledger"
+      | Fee_payer_not_permitted_to_send ->
+          "Fee payer account permissions don't allow sending funds with \
+           signature authorization"
```

Comment:
The fee payer always has a signature authorization, yes? This wording suggests that it's possible to use a different authorization which would allow sending funds, so maybe it's better not to mention the authorization.